newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/index.js

```diff
@@ -316,6 +331,52 @@ const GetStartedSection = ({
           </Line>
         </SectionRow>
       )}
+      {shouldShowInAppTutorialButtons && (
+        <SectionRow>
+          <Text size="title" noMargin>
+            <Trans>Mini tutorials</Trans>
+          </Text>
+          <Text size="body" color="secondary" noMargin>
+            <Trans>Learn game concepts in a few minutes.</Trans>
+          </Text>
+          <Line>
+            <div style={styles.bannerContainer}>
+              {inAppTutorialsFetchingError ? (
+                <PlaceholderError onRetry={fetchInAppTutorials}>
+                  <Trans>
+                    An error occurred when downloading the tutorials.
+                  </Trans>{' '}
+                  <Trans>
+                    Please check your internet connection or try again later.
+                  </Trans>
+                </PlaceholderError>
+              ) : inAppTutorialShortHeaders === null ? (
+                <PlaceholderLoader />
+              ) : (
+                <GridList
+                  cols={getColumnsFromWidth(windowWidth)}
+                  style={styles.grid}
+                  cellHeight="auto"
+                  spacing={ITEMS_SPACING * 2}
+                >
+                  {miniInAppTutorialCards.map(item => (
+                    <GridListTile key={item.key}>
+                      <InAppTutorialPhaseCard
+                        {...item}
+                        onClick={() =>
+                          selectInAppTutorial(
+                            PLINKO_MULTIPLIER_IN_APP_TUTORIAL_ID
```

Comment:
A bit weird to do a `miniInAppTutorialCards.map` and then hardcode the id, could it be in the item?

Reply:
yes definitely